packages/e2e-tests/specs/experiments/multi-entity-saving.test.js

```diff
@@ -122,13 +163,17 @@ describe( 'Multi-entity save flow', () => {
 
 		describe( 'Published state', () => {
 			it( 'Update button disabled after publish', async () => {
-				await publishPostWithPrePublishChecksDisabled();
+				await publishPost();
```

Comment:
Why was this changed?

Reply:
To properly publish the post now that I removed the unnecessary `disablePrePublishChecks`